tools/c7n_org/c7n_org/cli.py

```diff
@@ -36,6 +36,8 @@
 
 from c7n_org.utils import environ, account_tags
 
+from c7n_oci.constants import COMPARTMENT_IDS
```

Comment:
nack re direct import here, c7n-org maybe used without installing oci provider.

Reply:
Sure, I will use text instead of constants.